GitUI/BranchTreePanel/StashNode.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Diagnostics;
+using GitCommands;
+using GitUI.BranchTreePanel.Interfaces;
+using GitUI.CommandsDialogs;
+using GitUI.Properties;
+using GitUIPluginInterfaces;
+
+namespace GitUI.BranchTreePanel
+{
+    [DebuggerDisplay("(Tag) FullPath = {FullPath}, Hash = {ObjectId}, Visible: {Visible}")]
+    internal sealed class StashNode : BaseBranchNode
+    {
+        public StashNode(Tree tree, in ObjectId? objectId, string reflogSelector, string subject, bool visible)
+            : base(tree, reflogSelector.RemovePrefix("refs/"), visible)
```

Comment:
#10161 changed to display this in FormStash
![image](https://user-images.githubusercontent.com/6248932/196050239-73b4a397-a249-413e-99d5-35fa1ea34941.png)

The RevGrid show the complete ref (excluding refs/)
![image](https://user-images.githubusercontent.com/6248932/196050299-1cd3167f-d1bf-4568-a285-27db0d29cb9d.png)

Should the list present as in FormStash?
```suggestion
            : base(tree, reflogSelector.RemovePrefix("refs/").RemovePrefix("stash"), visible)
```
![image](https://user-images.githubusercontent.com/6248932/196050334-8795fdce-848a-4c79-aeea-66c4cbfcd9de.png)

Reply:
I have no strong opinion regarding the colon - not really needed but for consistency...

```suggestion
            : base(tree, reflogSelector.RemovePrefix(GitRefName.RefsStashPrefix), visible)
```

Reply:
Changed this, updated screenshot